src/modules/cluster/cluster-inventory-list/cluster-rbd-list.js

```diff
@@ -0,0 +1,29 @@
+/*
+    ============= How to use "clusterRbdList" directive ======
+            
+    - sample format:
+    <cluster-rbd-list></cluster-rbd-list>
+
+*/
+
+(function () {
+    "use strict";
+
+    var app = angular.module("TendrlModule");
+
+    app.directive("clusterRbdList", clusterRbdList);
```

Comment:
basically this should not be a component . it is not features related directive like , donut-chart or somthing .

this is just for redirection , Here you can see  , just i am redirecting to rbd controller and html view